task/git-clone/0.1/git-clone.yaml

```diff
@@ -188,6 +207,15 @@ spec:
       if [ "${EXIT_CODE}" != 0 ] ; then
         exit "${EXIT_CODE}"
       fi
+
+      if [ "${PARAM_ENABLE_SYMLINK_CHECK}" = "true" ] ; then
+        check_symlinks
+        EXIT_CODE="$?"
+        if [ "${EXIT_CODE}" != 0 ] ; then
+          exit "${EXIT_CODE}"
+        fi
```

Comment:
Isn't this behavior alredy enforced by
```
set -eu
```

Reply:
and because you are using pipe in your code, it should be probably extended to
```
set -euo pipefail
```

Reply:
Updated code and changed sh to bash so I can use process substitution.